tools/cli/workflow.go

```diff
@@ -364,3 +370,89 @@ func newActivityCommands() []cli.Command {
 		},
 	}
 }
+
+func newBatchCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:    "describe",
+			Aliases: []string{"desc"},
+			Usage:   "Describe a batch operation job",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagJobIDWithAlias,
+					Usage: "Batch Job ID",
+				},
+			},
+			Action: func(c *cli.Context) {
+				DescribeBatchJob(c)
+			},
+		},
+		{
+			Name:  "stop",
+			Usage: "Stop a batch operation job",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagJobIDWithAlias,
+					Usage: "Batch Job ID",
+				},
+				cli.StringFlag{
+					Name:  FlagReasonWithAlias,
+					Usage: "Reason to stop this batch job",
+				},
+			},
+			Action: func(c *cli.Context) {
+				StopBatchJob(c)
+			},
+		},
+		{
+			Name:    "list",
+			Aliases: []string{"l"},
+			Usage:   "Describe a batch operation job",
+			Flags: []cli.Flag{
+				cli.IntFlag{
+					Name:  FlagPageSizeWithAlias,
+					Value: 30,
+					Usage: "Result page size",
+				},
+			},
+			Action: func(c *cli.Context) {
+				ListBatchJobs(c)
+			},
+		},
+		{
+			Name:  "start",
+			Usage: "Start a batch operation job",
```

Comment:
I would add confirmation prompt that says something like:
You are about to terminate 100000 workflows. Are you sure?
n/y
and a flag to disable the confirmation

Reply:
Done